src/services/motion/types.ts

```diff
@@ -81,13 +81,13 @@ export const encodeMotionConfiguration = (
         obstacleDetector.setCamera(encodeResourceName(od.camera));
       }
       return obstacleDetector;
-    })
+    }) ?? new Array<pb.ObstacleDetector>()
   );
-  result.setPositionPollingFrequencyHz(obj.positionPollingFrequencyHz);
-  result.setObstaclePollingFrequencyHz(obj.obstaclePollingFrequencyHz);
-  result.setPlanDeviationM(obj.planDeviationM);
-  result.setLinearMPerSec(obj.linearMPerSec);
-  result.setAngularDegsPerSec(obj.angularDegsPerSec);
+  result.setPositionPollingFrequencyHz(obj.positionPollingFrequencyHz ?? 1);
```

Comment:
Are these necessary to coalesce? In other words, will this break somewhere if we do not send default values? My concern is that default values now potentially exist in multiple places (SDK and robot).

Reply:
>Are these necessary to coalesce? In other words, will this break somewhere if we do not send default values?... My concern is that default values now potentially exist - and would need to be maintained - in multiple places (SDK and robot).

Good catch! I will change this to be:
```
  if (obj.angularDegsPerSec) {
    result.setAngularDegsPerSec(obj.angularDegsPerSec);
  }
```